util/pkg/vfs/context.go

```diff
@@ -486,3 +491,42 @@ func (c *VFSContext) buildVaultPath(p string) (*VaultPath, error) {
 
 	return newVaultPath(c.vaultClient, scheme, u.Path)
 }
+
+func (c *VFSContext) buildAzureBlobPath(p string) (*AzureBlobPath, error) {
+	u, err := url.Parse(p)
+	if err != nil {
+		return nil, fmt.Errorf("failed to parse %q: %s", p, err)
+	}
+
+	if u.Scheme != "azureblob" {
+		return nil, fmt.Errorf("invalid Azure Blob schem: %q", p)
```

Comment:
Nit: scheme

Reply:
Ah thanks for catching! Will create a small PR to fix.